### ISSUE TITLE
关于损失函数的疑问

### ISSUE THREAD

**Issue by zsxzs** [VDIGPKU/RPAttack#6]
非常感谢您优秀的工作，看代码时我有一个关于损失函数的疑问，
loss += 1. - mseloss(det_confs, torch.zeros(det_confs.shape).cuda())
检测框置信度增加，损失是减小的，我应该怎么理解这个损失函数？
希望得到您的回复，谢谢。
